### ISSUE TITLE
Accessibility: The user is not informed if the filter category title accordion is open or closed.

### ISSUE THREAD

**Issue by MarkoTP** [microsoft-search/pnp-modern-search#4023]
**Version used**
4.13.1

**Describe the bug**
Accessibility:
A few PnP Modern Search issues were identified during the accessibility audit.
Using the PnP Modern Search Filters webpart with a screen reader (e.g., NVDA), the user is not informed if the filter category title accordion is open or closed.
WCAG 2.1, 4.1.2

**To Reproduce**
Detailed steps to reproduce the behavior:
Configure the search page as you like with search box, results, and filters web parts. There should be at least one filter defined.
Execute a search with a keyword that returns filterable results.
Press Tab when the focus is on the search box until you reach the filter category title.
Press Enter to open or close the filter category title accordion.

**Expected behavior**
The screen reader (e.g., NVDA) should inform the user whether the accordion is open or closed.

**Desktop (please complete the following information):**
Browser: Chrome, Version 129.0.6668.90 (Official Build) (64-bit)
